jll/G/GR_jll/Compat.toml

```diff
@@ -22,4 +22,4 @@ julia = "1.6.0-1"
 ["0.72.9-0"]
 FreeType2_jll = "2.10.4-2"
 Libtiff_jll = "4.5.1-4.5"
-Qt6Base_jll = "6.4.2-6"
+Qt6Base_jll = "6.4.2-6.5"
```

Comment:
This include 6.5.x series, is this what you want to do?

Reply:
No,I misunderstood how to translate the ~, I'll make it 6.4.